### ISSUE TITLE
you can't consume things if you're wearing the ninja hood

### ISSUE THREAD

**Issue by bastardblasted** [tgstation/tgstation#60876]
## Reproduction:
1)Take a dead ninja's helmet thing, put it on, realize you can't eat with it on and it's unremovable
2) cry

**Comment by Cookhimself** [comment]
sugar patches

**Comment by NanoCats** [comment]
I feel like it'd be fair for the ninja to either have a nutrient pump, or

- The ninja's shit shouldn't be filled with glue, you should be able to take it off, no more no drop once the ninja dies
- The ninja should just be able to eat through his mask because fuck it why not, the clown can

**Comment by Sealed101** [comment]
ninjas can remove their suit after a powerimg down delay to eat/inject/drink/run naked. i think it could be okay to make it so the hood doesnt block eating or drinking attempts tho

**Comment by timothymtorres** [comment]
https://github.com/tgstation/tgstation/blob/55b92786dcf0f9c5172efd111a6ecbf83ec0dae2/code/modules/ninja/suit/head.dm#L9-L18

So the ninja mask is a subtype of a regular gas mask that inherits the following flags:

https://github.com/tgstation/tgstation/blob/72dce8796efc355d142a46b9d74234df60115d46/code/modules/clothing/masks/gasmask.dm#L1-L11

This is the clown mask for reference.

https://github.com/tgstation/tgstation/blob/72dce8796efc355d142a46b9d74234df60115d46/code/modules/clothing/masks/gasmask.dm#L150-L158

The solution would be to change `flags_cover = MASKCOVERSEYES | MASKCOVERSMOUTH | PEPPERPROOF` for the ninja mask to just be `flags_cover = MASKCOVERSEYES | PEPPERPROOF` but I have a feeling there could be some side effects.  This might make the ninja suspectible to pepper & smoke grenades or any kind of chemical sprays.

Overall I don't think eating or drinking with the hood is the real problem.  If the mask and equipment is behaving like it's cursed and cannot be removed then that is the real bug.  The relevant code for this is here:

https://github.com/tgstation/tgstation/blob/4e2a36fe7e56fad93495adf01c697f4b5de04e90/code/modules/ninja/suit/suit.dm#L176-L213

Basically the ninja hood should be removable for non-ninjas and @bastardblasted should update his title.

**Comment by timothymtorres** [comment]
Duplicate of #48868
